x-pack/plugin/vector-tile/src/javaRestTest/java/org/elasticsearch/xpack/vectortile/VectorTileRestIT.java

```diff
@@ -405,7 +404,7 @@ public void testCentroidGridTypeNameCollision() {
                 + "}"
         );
         ResponseException ex = expectThrows(ResponseException.class, () -> execute(mvtRequest));
-        assertThat(ex.getMessage(), Matchers.containsString("Invalid aggregation name [[centroid]]"));
+        assertThat(ex.getMessage(), Matchers.containsString("Invalid aggregation name [_mvt_name]"));
```

Comment:
Maybe add an explanation why it is invalid (like aggregations cannot start with `_mvt`)